docs/source/metric.rst

```diff
@@ -95,29 +81,20 @@ as:
 
     where :math:`r_s=2*G*M/c^2`
 
-and is called the Schwarzschild Radius, a point beyond where space and time flips 
-and any object inside the radius would require speed greater than speed 
-of light to escape singularity, where the curvature of space becomes infinite and 
-so is the case with the tidal forces. Putting :math:`r=\infty`, we see that the metric 
-transforms to a metric for a flat space defined by spherical coordinates. 
+and is called the Schwarzschild Radius, a point beyond where space and time flips and any object inside the radius would require speed greater than the speed of light to escape singularity, where the curvature of space becomes infinite and so is the case with the tidal forces. Putting :math:`r=\infty`, we see that the metric transforms into a metric for a flat space defined by spherical coordinates. 
 
 :math:`\tau` is the proper time, the time experienced by the particle in motion in 
-the space-time while :math:`t` is the coordinate time obeserved by an observer 
+the space-time while :math:`t` is the coordinate time observed by an observer 
```

Comment:
Good observation! I would be including it with PR #186 and would make you the author of the commit